tls/s2n_config.c

```diff
@@ -1067,3 +1070,26 @@ int s2n_config_set_recv_multi_record(struct s2n_config *config, bool enabled)
 
     return S2N_SUCCESS;
 }
+
+/* Indicates if the connection should attempt to enable kTLS. */
```

Comment:
Not really sure this comment is accurate. The function enables ktls, it doesn't indicate anything.